src/actions/graph.js

```diff
@@ -93,10 +80,12 @@ const chooseLayout = (instance, state) =>
 const layoutWorker = preventWorkerQueues(worker, chooseLayout);
 
 /**
- * Formula to determine if the pipeline is chonky
+ * Formula to determine if the pipeline is large, which is simply the sum of
+ * both the amount of nodes and edges within the graph. Edges are given a
+ * strongger weight of 1.5 with it being more computational heavy to render.
```

Comment:
I was thinking more that this should explain the goal rather than describing the solution? e.g. "Estimate the point at which the graph will take unreasonably long to render in terms of input nodes and edges". Also looks like a spelling issue on `strongger`.

Reply:
updated with new wording as well as descriptions of params